api/src/main/resources/workbench.yaml

```diff
@@ -850,7 +851,8 @@ definitions:
       subjects to include or exclude from the cohort.
     type: object
     required:
-      - include
+      - includes
```

Comment:
Are both of these actually required? (What if you don't exclude anyone?)

Is there a separate mechanism for getting a total count for the CDR to display somewhere?

Reply:
i made them required because the swagger code gen creates the collections as null if they aren't required like so:
@JsonProperty("excludes")
  private List<SearchGroup> excludes = null;

This causes me to do null checks before iterating my collection. Require actually makes swagger code gen initialize the collections like so:
@JsonProperty("excludes")
  private List<SearchGroup> excludes = new ArrayList<SearchGroup>();

on the UI side we just pass a json search request like so:
{
      includes: [
        {items: [searchGoupItem.toJS()]}
      ],
      excludes: []
    }